onnxruntime/onnxruntime-engine/src/main/java/ai/djl/onnxruntime/engine/OrtNDManager.java

```diff
@@ -105,13 +105,23 @@ public NDList load(Path path) {
     /** {@inheritDoc} */
     @Override
     public NDArray zeros(Shape shape, DataType dataType) {
-        throw new UnsupportedOperationException("Not supported for ONNX Runtime");
+        int bytes = dataType.getNumOfBytes();
+        int size = Math.toIntExact(bytes * shape.size());
+        ByteBuffer buffer = allocateDirect(size);
+        return create(dataType.asDataType(buffer), shape, dataType);
     }
 
     /** {@inheritDoc} */
     @Override
     public NDArray ones(Shape shape, DataType dataType) {
-        throw new UnsupportedOperationException("Not supported for ONNX Runtime");
+        int bytes = dataType.getNumOfBytes();
+        int size = Math.toIntExact(bytes * shape.size());
+        ByteBuffer buffer = allocateDirect(size);
+        for (int i = 0; i < size; ++i) {
+            buffer.put((byte) 1);
```

Comment:
Is this right? Wouldn't it put the same bytes regardless of the data type?

Reply:
Fixed.